content.md

```diff
@@ -76,7 +76,6 @@ This list is also available online at: https://resources.osucord.moe/
 ### Skins
 - [osuck](https://skins.osuck.net/) - Website to get skins for all game modes
 - [Skinship's skin compendium](https://compendium.skinship.xyz/) - Easy to search archive of all osu! skins posted in the Completed Skins forum
```

Comment:
```suggestion
- [Skinship's skin compendium](https://compendium.skinship.xyz/) - Easy to search archive of all osu! skins posted in the Completed Skins forum
- [Circle People](https://circle-people.com/skins/) - List of skins used by various well-known players
```

Reply:
CC @Ruben172

Reply:
nothing changed from my suggested change lmao

Reply:
I know but it the diff doesn't look as nasty :upside_down_face: